### PR TITLE
add travis file

### PR DESCRIPTION
**Pull request by arthru**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  - sudo pip install unidecode
+  - sudo pip install git+https://github.com/akretion/prestapyt.git#egg=prestapyt
+
+install:
+  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  - travis_install_nightly
+
+script:
+  - travis_run_flake8
```
Comment (pedrobaeza) [inline .travis.yml]: This is not needed in current version. It's done in the other call

Comment (arthru) [inline .travis.yml]: sorry, I don't get your comment. Is the travis_run_flake8 directive not needed ?

Comment (arthru) [inline .travis.yml]: ok, I now understand, sorry

**Comment by arthru** [comment]
Ok, so, at this point, travis build fail, but at least, we install dependencies and test_server is green...

**Comment by pedrobaeza** [comment]
Thanks :+1: 

We can let fix PEP8 for another PR.

**Comment by yvaucher** [comment]
:+1: Thanks

**Comment by bwrsandman** [comment]
Hey. Looks like you forgot to add the README badges. Could you add those as well?

**Pull request merged**